Waiter.py

```diff
@@ -0,0 +1,104 @@
+import os
+import re
+import time
+from colorama import Fore, Back, Style
+import pandas as pd
+from tabulate import tabulate as tb
+import numpy as np
+
+
+
+#Menu Options
+print("[ORDER & PAY HERE]")
+NumCust = input("How many of you will be dining with us today?")
+
+
+#Trying to figure out how to clear out the output to make output cleaner for user
+# def ClearOutput():
+#     if os.name == 'nt':
+#         os.system('cls')
+#     else:
+#         os.system('clear')
+# ClearOutput()
+
+#Item Details: Name & Prices ==> Just edit "ItemDet" to make future amendments
+ItemDet = {'Cheese Burger':6, 'Fries':3, 'Tenders': 4, 'Soda':2}
+OrderList = list(ItemDet.keys())
+ItemP = list(ItemDet.values())
```

Comment:
Just write all the variable names out. Your editor should help you with auto complete anyway.